sycl/include/CL/sycl/detail/pi.h

```diff
@@ -53,27 +53,31 @@ typedef pi_uint64   pi_bitfield;
 // TODO: populate PI enums.
 //
 typedef enum {
-  PI_SUCCESS                      = CL_SUCCESS,
-  PI_RESULT_INVALID_KERNEL_NAME   = CL_INVALID_KERNEL_NAME,
-  PI_INVALID_OPERATION            = CL_INVALID_OPERATION,
-  PI_INVALID_KERNEL               = CL_INVALID_KERNEL,
-  PI_INVALID_QUEUE_PROPERTIES     = CL_INVALID_QUEUE_PROPERTIES,
-  PI_INVALID_VALUE                = CL_INVALID_VALUE,
-  PI_INVALID_CONTEXT              = CL_INVALID_CONTEXT,
-  PI_INVALID_PLATFORM             = CL_INVALID_PLATFORM,
-  PI_INVALID_DEVICE               = CL_INVALID_DEVICE,
-  PI_INVALID_BINARY               = CL_INVALID_BINARY,
-  PI_INVALID_QUEUE                = CL_INVALID_COMMAND_QUEUE,
-  PI_OUT_OF_HOST_MEMORY           = CL_OUT_OF_HOST_MEMORY,
-  PI_INVALID_PROGRAM              = CL_INVALID_PROGRAM,
-  PI_INVALID_MEM_OBJECT           = CL_INVALID_MEM_OBJECT,
-  PI_OUT_OF_RESOURCES             = CL_OUT_OF_RESOURCES,
-  PI_INVALID_EVENT                = CL_INVALID_EVENT,
-  PI_INVALID_EVENT_WAIT_LIST      = CL_INVALID_EVENT_WAIT_LIST,
+  PI_SUCCESS = CL_SUCCESS,
+  PI_RESULT_INVALID_KERNEL_NAME = CL_INVALID_KERNEL_NAME,
```

Comment:
May we align this code spelling with other codes? I mean
```suggestion
  PI_INVALID_KERNEL_NAME = CL_INVALID_KERNEL_NAME,
```